extensions/wikia/PortableInfobox/tests/PortableInfoboxRenderServiceHelperTest.php

```diff
@@ -415,41 +417,47 @@ public function customWidthProvider() {
 				'preferred' => 300,
 				'result' => [ 'width' => 300, 'height' => 200 ],
 				'thumbnail' => [ 'width' => 300, 'height' => 200 ],
+				'thumbnail2x' => [ 'width' => 600, 'height' => 400 ],
 				'original' => [ 'width' => 300, 'height' => 200 ]
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "original" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)